### PR TITLE
[Bug 1163855] Make helptext unicode for Profile fields

### PR DESCRIPTION
**Pull request by mythmon**
Because apparently this matters a lot to... something? It was causing tracebacks in the API on prod. I updated the tests to find this kind of error, though I don't really understand it.

r?

### CODE REVIEW

**Comment by mythmon** [comment]
PS: This is currently broken in prod due to today's push. It probably effects BuddyUp, and a few power-user contributors.

File: kitsune/users/migrations/0006_auto_update_help_text_email_sent_fields.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+    dependencies = [
+        ('users', '0005_set_initial_contrib_email_flag'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='profile',
+            name='first_l10n_email_sent',
+            field=models.BooleanField(default=False, help_text='Has been sent a first revision contribution email.'),
```
Comment (willkg) [inline kitsune/users/migrations/0006_auto_update_help_text_email_sent_fields.py]: You changed the help text on two boolean fields, but this is still a string and not unicode and there's only one boolean field represented. That seems fishy.

Comment (mythmon) [inline kitsune/users/migrations/0006_auto_update_help_text_email_sent_fields.py]: Oooops. I did some surgery on this migration, and in my haste cut too deeply.

**Comment by mythmon** [comment]
Added the second migration.

**Comment by willkg** [comment]
I didn't run this or test it, but it looks good to me. r+

**Comment by willkg** [comment]
I should run the tests. Give me ten minutes.

**Comment by willkg** [comment]
I get one test failure:

```
======================================================================
FAIL: kitsune/dashboards/tests/test_cron.py:L10nMetricsTests.test_update_l10n_coverage_metrics
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/willkg/mozilla/kitsune/kitsune/dashboards/tests/test_cron.py", line 281, in test_update_l10n_coverage_metrics
    eq_(4, WikiMetric.objects.filter(locale='es').count())
  File "/home/willkg/.virtualenvs/kitsune/lib/python2.7/site-packages/nose/tools.py", line 31, in eq_
    assert a == b, msg or "%r != %r" % (a, b)
AssertionError: 4 != 6
```

But that could be spurious. I occasionally get test errors and then they "go away".

**Comment by willkg** [comment]
Also :red_herring: should be an emoji.

**Comment by willkg** [comment]
On second thought, that test is failing all the time--not sporadically. It's weird that that's related, though, right?

**Comment by mythmon** [comment]
Uh oh. I think that is a separate issue from another bug. I suspect that if you run the tests on master, you'll get the same problem. I'll double check that myself, and dig into why we have that test failure.

**Comment by mythmon** [comment]
Ok, I found the problem. There was a test that wasn't updated for the metrics dashboard top 100 bug. I updated it and pushed again. Now all the tests are passing for me.

Also, :red_herring: should totally be an emoji.

**Comment by willkg** [comment]
Works for me! r+

**Pull request merged**